regression/tests/common/test_html_component.py

```diff
@@ -79,6 +77,11 @@ def test_unit_studio_view(self):
         self.lms_courseware.go_to_section(section_name, subsection_name)
         # View unit in the studio
         self.lms_courseware.view_unit_in_studio()
+        # View the course in studio.
+        self.studio_course_outline.visit()
```

Comment:
Do you need the `visit()` call?  I thought the test was ending up on the right page but just not expanding the appropriate subsection and unit; `visit()` takes the browser to this page even if it's currently on a different page entirely.

Reply:
`self.lms_courseware.view_unit_in_studio()` loads this page and you can see sections are appearing as bread-crumbs and its not possible to click them.

<img width="951" alt="Screen Shot 2021-03-25 at 8 05 36 PM" src="https://user-images.githubusercontent.com/445320/112497847-cccd6600-8da7-11eb-9cb3-045c2ed35a5a.png">

------------------------------------------------------------------------------------------------------------
but `self.studio_course_outline.visit()` loads same page in collapsable form.


<img width="847" alt="Screen Shot 2021-03-25 at 8 07 14 PM" src="https://user-images.githubusercontent.com/445320/112498250-27ff5880-8da8-11eb-991e-fb8d92a6c4ac.png">

Reply:
This is verifying the locators.
```
self.assertEqual(
            get_data_locator(self.unit_container_page),
            data_locator, 'Correct component is opened'
        )
```